fvm/env.go

```diff
@@ -283,6 +284,75 @@ func storageMBUFixToBytesUInt(result cadence.Value) uint64 {
 	return result.ToGoValue().(uint64) / 100
 }
 
+func (env *commonEnv) GetStorageCapacity(
+	address common.Address,
+) (
+	value uint64,
+	err error,
+) {
+	defer env.StartSpanFromRoot(trace.FVMEnvGetStorageCapacity).End()
+
+	err = env.Meter(meter.ComputationKindGetStorageCapacity, 1)
+	if err != nil {
+		return 0, fmt.Errorf("get storage capacity failed: %w", err)
```

Comment:
Does `Meter` actually get the storage capacity or just increment the meter by 1?

If is later, it's better to mention that it's the metering failed not the actual data query failed

```suggestion
		return 0, fmt.Errorf("meter get storage capacity failed: %w", err)
```

Reply:
the error's detail is specify at the end.  for metering error, the specific string is "computation limit exceed ...".  It should be sufficiently clear.